src/index.tsx

```diff
@@ -721,6 +739,39 @@ class App extends React.Component<{}, AppState> {
     }
   };
 
+  private deleteSelectedElements = () => {
+    deleteSelectedElements();
+    this.forceUpdate();
+  };
+
+  private moveToFront = () => {
+    const elementsWithIndexes = getSelectedElementsWithIndexes();
+
+    elementsWithIndexes.forEach(elementWithIndex => {
+      elements.splice(elementWithIndex.index, 1);
+    });
+
+    elementsWithIndexes.forEach(elementWithIndex => {
+      elements.push(elementWithIndex.element);
+    });
+
+    this.forceUpdate();
+  };
+
+  private sendToBack = () => {
+    const elementsWithIndexes = getSelectedElementsWithIndexes();
+
+    elementsWithIndexes.forEach(elementWithIndex => {
+      elements.splice(elementWithIndex.index, 1);
+    });
+
+    elementsWithIndexes.forEach(elementWithIndex => {
+      elements.splice(0, 0, elementWithIndex.element);
```

Comment:
This is the problem you described. If you have two elements selected [A, B], then you're going to push them in the reverse order [B, A]. So they will be "swapped"